components/uuf-core/src/main/java/org/wso2/carbon/uuf/internal/deployment/AppCreator.java

```diff
@@ -255,26 +256,15 @@ private void addBindings(List<ComponentConfig.Binding> bindingEntries, Bindings
     }
 
     private void addAPIs(List<ComponentConfig.API> apis, String appContextPath, String componentContextPath,
-                         String componentName, ClassLoader classLoader) {
+                         ClassLoader classLoader) {
         if ((apis == null) || apis.isEmpty()) {
             return;
         }
 
         for (ComponentConfig.API api : apis) {
-            String className = api.getClassName();
-            String uri = appContextPath + componentContextPath + "/apis" + api.getUri();
-            Object apiImplementation;
-            try {
-                apiImplementation = classLoader.loadClass(className).newInstance();
-            } catch (InstantiationException | IllegalAccessException | ClassNotFoundException e) {
-                throw new UUFException(
-                        "Cannot deploy REST API '" + className + "' for component '" + componentName + "'.", e);
-            }
-            Dictionary<String, String> serviceProperties = new Hashtable<>();
-            serviceProperties.put("contextPath", uri);
-            classLoaderProvider.deployAPI(apiImplementation, serviceProperties);
-            LOGGER.info("Deployed REST API '{}' for component '{}' with context path '{}'.", className, componentName,
-                        uri);
+            String apiContextPath = appContextPath + componentContextPath + "/apis" + api.getUri();
```

Comment:
Since this string is not used anywhere else, and it is self-explanatory, I don't think it should be put to a constant.